packages/components/src/flyout/flyout/README.md

```diff
@@ -0,0 +1,25 @@
+# Flyout
+
+<div class="callout callout-alert">
+This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
+</div>
+
+`Flyout` is a component to render a floating content modal. It is similar in purpose to a tooltip, but renders content of any sort, not only simple text.
+
+## Usage
+
+```jsx
+import { Button, __experimentalFlyout as Flyout, View, Text } from '@wordpress/components';
```

Comment:
```suggestion
import { Button, __experimentalFlyout as Flyout, __experimentalText as Text } from '@wordpress/components';
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32197/commits/77b5734b35b36164c217029c347363391cf69331